analyses/compare-gistic/util/GISTIC-comparison-functions.R

```diff
@@ -215,62 +202,69 @@ plot_genes_venn_diagram_wrapper <- function(cohort_genes_file,
 }
 
 #### Implemented in `02-GISTIC-tidy-data-prep.Rmd` ----------------------------
-
 prepare_gene_level_gistic <- function(all_lesions_file,
                                       amp_genes_file,
                                       del_genes_file,
-                                      output_filename) {
+                                      gene_mapping_filepath,
+                                      gene_status_filepath,
+                                      residual_q_threshold = 1) {
 
   # Given the file paths to GISTIC's `all_lesion.conf_90.txt`,
   # `amp_genes.conf_90.txt`, and `del_genes.conf_90.txt` files,
-  # read in and tidy this data into a data.frame that contains
-  # the amplified/deleted genes, the detection peak, and
-  # the IDs of the samples in the corresponding peaks along with
-  # with their CN status call.
+  # read in and tidy this data into a two data.frames that are written to file.
   #
   # Args:
   #   all_lesions_file: file path to GISTIC's `all_lesion.conf_90.txt` file
   #   amp_genes_file: file path to GISTIC's `amp_genes.conf_90.txt` file
   #   del_genes_file: file path to GISTIC's `del_genes.conf_90.txt` file
-  #   output_filename: string to save the output file as
+  #   gene_mapping_filepath: TSV file path for the first data.frame that
```

Comment:
```suggestion
  #   gene_mapping_filepath: file path for output TSV which has these columns: 
```
If this isn't what you mean correct me.

Reply:
My wording isn't great, but I just want to clarify that these are files that don't currently exist but will be made by this function. ?